README.md

```diff
@@ -25,9 +25,11 @@ Second, go to src folder in your workspace:
 ```
 cd ~/catkin_ws/src
 ```
-Third,clone gilbreth_gazebo and gilbreth_description folder:
+Third, download the Gilbreth and dependent packages:
 ```
-git clone https://github.com/swri-robotics/gilbreth.git
+wstool init .
+wstool merge https://github.com//gilbreth/kinetic-devel/gilbreth.rosinstall
+wstool update
```

Comment:
Should there be a ```rosdep``` step.  Typically running
```
rosdep install --from-paths . --ignore-src -r -y
```
installs all the debian dependencies.

Reply:
Personal, but I never add `-r`: `rosdep` will 'ignore' any keys it can't resolve, which sounds nice, but ni the end you have a system that still can't build your workspace.

In my experience, unless people get an error message and a process *stops* without completing, they'll assume everything is ok ("why did it otherwise ignore the error?").

Reply:
@gavanderhoorn yes, minor oversight on my part.  I just copied and pasted from a wiki page somewhere.